app/views/pages/scholarships.html.erb

```diff
@@ -5,11 +5,15 @@
         <p class="lead">At Operation Code, we partner with both code schools and industry leaders to make transitioning to software development less painful and accessible for our nation's finest and their families.</p>
       </div>
 
-      <div class="center">
-        <iframe src="https://player.vimeo.com/video/125189545" width="700" height="481" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+
+      <div class="row center">
+        <div class="video-container">
+          <iframe src="https://player.vimeo.com/video/125189545" width="700" height="481"
+            frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen>
+          </iframe>
+        </div>
       </div>
-      <blockquote>Former Air National Guard soldier, Nuatu Tseggai, benefited from one of our scholarships to attend Ruby on Ales. An Epicodus alumn, Nuatu currently works as a site reliability engineer at New Relic.</blockquote>
-    </div>
+    <blockquote>Former Air National Guard soldier, Nuatu Tseggai, benefited from one of our scholarships to attend Ruby on Ales. An Epicodus alumn, Nuatu currently works as a site reliability engineer at New Relic.</blockquote>
```

Comment:
I'm running this branch locally, the video doesn't even show on my screen unless `class="col m12"` on line 3 is removed.   Not sure if that column class is needed - divs should span across full-screen by default.  Running Chrome on Mac.